fabulaws-config.yml

```diff
@@ -70,7 +70,7 @@
     main: -c 10
 
 # Start this many Gunicorn workers for each CPU core
-  gunicorn_worker_multipler: 8
+  gunicorn_worker_multipler: 4
```

Comment:
Is this supposed to be `gunicorn_worker_multipler` or `gunicorn_worker_multiplier`?